### ISSUE TITLE
Publish useable alpha

### ISSUE THREAD

**Issue by espen42** [enonic/lib-frontend-proxy#5]
v0.1.0 should be available from repo.enonic.com, but attempts to publish fails.

**Comment by espen42** [comment]
```
./gradlew publish --info

(...)

> Task :publishMavenJavaPublicationToPublicRepository FAILED
Caching disabled for task ':publishMavenJavaPublicationToPublicRepository' because:
  Build cache is disabled
Task ':publishMavenJavaPublicationToPublicRepository' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Publishing to repository 'public' (https://repo.enonic.com/public)
Resource missing. [HTTP GET: https://repo.enonic.com/public/com/enonic/lib/lib-frontend-proxy/0.1.0-SNAPSHOT/maven-metadata.xml]
Uploading maven-metadata.xml to /public/com/enonic/lib/lib-frontend-proxy/0.1.0-SNAPSHOT/maven-metadata.xml
Uploading lib-frontend-proxy-0.1.0-20211117.134122-1.jar to /public/com/enonic/lib/lib-frontend-proxy/0.1.0-SNAPSHOT/lib-frontend-proxy-0.1.0-20211117.134122-1.jar
:publishMavenJavaPublicationToPublicRepository (Thread[Execution worker for ':' Thread 5,5,main]) completed. Took 1.213 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':publishMavenJavaPublicationToPublicRepository'.
> Failed to publish publication 'mavenJava' to repository 'public'
   > Could not PUT 'https://repo.enonic.com/public/com/enonic/lib/lib-frontend-proxy/0.1.0-SNAPSHOT/lib-frontend-proxy-0.1.0-20211117.134122-1.jar'. Received status code 409 from server: Conflict
```

See also https://repo.enonic.com/public/com/enonic/lib/lib-frontend-proxy/

**Comment by rymsha** [comment]
0.1.0-SNAPSHOT won't land on public repo. Only release versions are accepted there

**Comment by espen42** [comment]
Published [0.1.0-RC1](https://repo.enonic.com/public/com/enonic/lib/lib-frontend-proxy/0.1.0-RC1/).
